Automattic/wp-calypso
# Update Site Importer visuals and activate in all ENVs

This PR adds a branding to site parser and activates it.

To test:
- My Sites
- pick a testing site or make a new one
- Settings
- Import

Make sure it looks good and you don't see any regressions or visual glitches.
